packages/staking/src/features/overview/Overview.tsx

```diff
@@ -103,12 +100,9 @@ export const Overview = () => {
           status={currentPortfolio.length === 1 ? 'simple-delegation' : 'multi-delegation'}
         />
       </Box>
-      {stakingNotification && (
+      {stakingNotifications && (
```

Comment:
```suggestion
      {stakingNotifications.length > 0 && (
```

otherwise and empty div will be added as `[]` evaluates to true